var/spack/repos/builtin/packages/py-pyfr/package.py

```diff
@@ -44,8 +50,9 @@ class PyPyfr(PythonPackage):
     depends_on("py-pytools@2016.2.1:", type=('build', 'run'))
     depends_on("py-scipy", type=('build', 'run'))
 
-    # Optional  dependecies
+    # Optional dependecies
     depends_on("metis@5.0:", when="+metis", type=('run'))
     depends_on("scotch@6.0:", when="+scotch", type=('run'))
     depends_on("cuda@8.0:", when="+cuda", type=('run'))
     depends_on("rocblas@4.5.0:", when="+hip", type=('run'))
+    depends_on("libxsmm@master+shared +noblas CODE_BUF_MAXSIZE=262144", when="+libxsmm", type=('run'))
```

Comment:
```suggestion
    depends_on("libxsmm@1.18:+shared ~blas CODE_BUF_MAXSIZE=262144", when="+libxsmm", type=('run'))
```
Then you don't have to update once 1.18 is released

Reply:
done, thanks